drf_spectacular/views.py

```diff
@@ -81,12 +82,23 @@ class SpectacularSwaggerView(APIView):
 
     @extend_schema(exclude=True)
     def get(self, request, *args, **kwargs):
+        swagger_bundle_js_href = "{}/swagger-ui-bundle.js".format(spectacular_settings.SWAGGER_UI_DIST)
```

Comment:
not necessary. just pass in `dist` and `favicon_href` in the template below and we are good. you can append strings like `/swagger-ui.css"` in the template